src/childinfo.c

```diff
@@ -31,9 +31,8 @@
 #include <unistd.h>
 
 typedef struct {
-    int process_type;           /* AOF or RDB child? */
-    int on_exit;                /* COW size of active or exited child */
-    size_t cow_size;            /* Copy on write size. */
+    int information_type;            /* Type of information */
```

Comment:
```suggestion
    childInfoType information_type;            /* Type of information */
```

If we have an enum, let's use it everywhere instead of `int`.
Also considering a shortening, `type` is as readable as `information_type` IMHO.